docs/source-pytorch/data/iterables.rst

```diff
@@ -0,0 +1,93 @@
+:orphan:
+
+Arbitrary iterable support
+==========================
```

Comment:
Any reason why these tiles aren't at the highest level? Probably should be

```suggestion
##########################
Arbitrary iterable support
##########################
```

And then the levels below with `*****`. 
See comment below

Reply:
I coped the structure used in `docs/source-pytorch/accelerators/gpu_basic.rst`

I'm not sure what's the difference. Is this wrong?

Feel free to push edits for this!

Reply:
Synced offline: Adrian said he'd look into this in a follow-up